_pages/05-tests.md

```diff
@@ -2,64 +2,64 @@
 layout: page
 permalink: /tests/
 title: "Tests"
+page-title: "M-Lab Tests"
 menu-item: true
-grid-quick-links: true
+breadcrumb: tests
 grid-section:
-  - group-heading: "Performance"
+  - group-heading: "Current Tests"
     group-description: "Performance tests measure basic and advanced performance characteristics of your network, such as speed, latency, jitter, and much more."
     group-list-class: "l-grid l-3col"
     grid-items:
       - link: "/tests/ndt/"
         image: "ndt.png"
         heading: "NDT (Network Diagnostic Tool)"
         desc: "Tests your connection speed, and provides a sophisticated diagnosis of problems limiting speed."
+        categories: "performance"
       - link: "/tests/npad/"
         image: "npad.png"
         heading: "NPAD (Network Path &amp; Application Diagnostics)"
         desc: "Diagnoses common problems that impact last-mile broadband networks."
+        categories: "performance"
       - link: "/tests/neubot/"
         image: "neubot.png"
         heading: "Neubot"
         desc: "Performs periodic tests to measure network performance and traffic throttling."
+        categories: "performance"
       - link: "/tests/bismark/"
         image: "neubot.png"
```

Comment:
Noted for a future update. This could be an opportunity to ask tool developers about a preferred image for their test?